README.md

```diff
@@ -295,6 +295,11 @@ Look for a token in the session and verify it
 
 Look for a token in cookies and exchange it for an access token
 
+#### `Guardian.Plug.SlidingCookie`
+
+Replace a token in cookies with a new one if it is older than a configured
```

Comment:
I am thinking this sounds a bit backward. Would it make sense that, what we configure is the minimum remaining time of the token, before we would refresh it? For me, this is mostly the way I think about refresh time 

I think the current way can cause some trouble if you change the ttl of the refresh token.
Say we have a refresh ttl of 7 days, and we want to refresh only when there is one day left, we would set the sliding ttl to 6 days. Changing the refresh ttl to 14 would cause the refresh to happen when there are still 8 days left.  

@mwri  @ueberauth/developers  Let me hear what you think

Reply:
I'm absolutely with you on this, I tossed up about which way to go to be honest, a time after which it's renewed vs a time relative to the expiry... I'm very happy to change it to a minimum time remaining instead.

Reply:
I think that would make sense :D